### PR TITLE
Adapt for local run

### PR DESCRIPTION
**Pull request by MonikaFu**
* Add R-project 
* correct default directories to point to one level up from current dir

### CODE REVIEW

File: main.R [hunk 1]
```diff
@@ -139,9 +139,9 @@ techexposure_data
 
 # input and output directories -------------------------------------------------
 
-input_dir <- "./inputs"
-output_dir <- "./outputs"
-data_dir <- "./data"
+input_dir <- "../inputs"
+output_dir <- "../outputs"
+data_dir <- "../data"
```
Comment (jdhoffa) [inline main.R:145]: The original was correct. The expectation for file I/O on an RStudio run is that `inputs`, `data` and `outputs` directories are nested within the repo itself. This is critical to ensure that the containerization functions as expected, see:

https://github.com/RMI-PACTA/workflow.pacta.dashboard/blob/320b90a76677bcc372650c356aa3c7cccc0d3aad/docker-compose.yml#L8
https://github.com/RMI-PACTA/workflow.pacta.dashboard/blob/320b90a76677bcc372650c356aa3c7cccc0d3aad/docker-compose.yml#L12
https://github.com/RMI-PACTA/workflow.pacta.dashboard/blob/320b90a76677bcc372650c356aa3c7cccc0d3aad/docker-compose.yml#L16 

Although, I realize now that we should probably add those directories to the `.gitignore` as well.

Dealt with in https://github.com/RMI-PACTA/workflow.pacta.dashboard/pull/16

**Review by jdhoffa — CHANGES_REQUESTED** [review]

File: .Rhistory
Comment (jdhoffa) [inline .Rhistory:1]: Please `.gitinore` this file

**Review by jdhoffa — CHANGES_REQUESTED** [review]

File: main.R (hunk 1 above)
Comment (MonikaFu) [reply main.R:145]: ok, now it makes sense. I didn't see them in .gitignore that is why I was confused about the structure.

Comment (jdhoffa) [reply main.R:145]: I just added them!! No problem :)

**Comment by MonikaFu** [comment]
@jdhoffa changes you requested are implemented.

**Review by jdhoffa — APPROVED** [review]

**Pull request merged**